doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md

```diff
@@ -57,22 +57,37 @@ It must be said [PostgreSQL only accepts](https://www.postgresql.org/docs/curren
 
 PostgreSQL [databases name length](http://www.postgresql.org/docs/current/static/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS) is limited to 63 characters.
 
+Since version 2.2.0 It's added a new capability for Cygnus to create the schema name on runtime, this is possible trough enabling a specific Data Model on agent properties. That works the following way.
+
+* Data model by entity database (`data_model=dm-by-entity-database`). For this datamodel the name of the database will be auto generated by the sink, this name will be the `fiware-service` found on the headers of the first request incoming to the sink.
+* Data model by entity database schema (`data_model=dm-by-entity-database-schema`). For this datamodel the name of the database will be auto generated by the sink, this name will be the `fiware-service` found on the headers of the first request incoming to the sink.
```

Comment:
Both bullets seems to be the same... is that correct?

Reply:
Moreover... currently there is a configuration parameter in postgis/postgresql sink to specify the database. What happens in this case? The configuration setting is ignored? The behaviour be specified in the documentation (both in this section and in the database parameter in the configuration table)

Reply:
It was on purpose that those bullets to be the same because for now they do the same.

About the second comment.
Fixed in [9bcedc4](https://github.com/telefonicaid/fiware-cygnus/pull/1876/commits/9bcedc4805e6b3e4e48b7e5a357e18931cd62a0a)